packages/web-components/fast-components/src/tree-item/tree-item.styles.ts

```diff
@@ -95,232 +96,174 @@ export const treeItemStyles: (
-    }
-
-    :host(:${focusVisible}) .positioning-region {
-        border: ${focusStrokeOuter} calc(${strokeWidth} * 1px) solid;
-        border-radius: calc(${controlCornerRadius} * 1px);
-        color: ${neutralForegroundRest};
+      outline: none;
     }
-
     .positioning-region {
-        display: flex;
-        position: relative;
-        box-sizing: border-box;
-        border: transparent calc(${strokeWidth} * 1px) solid;
-        height: calc((${heightNumber} + 1) * 1px);
+      display: flex;
+      position: relative;
+      box-sizing: border-box;
+      background: ${neutralFillStealthRest};
+      border: calc(${strokeWidth} * 1px) solid transparent;
+      border-radius: calc(${controlCornerRadius} * 1px);
+      height: calc((${heightNumber} + 1) * 1px);
+    }
+    :host(:not([disabled])) .positioning-region:hover {
+      background: ${neutralFillStealthHover};
+      color: ${neutralForegroundHover};
+    }
+    :host(:not([disabled])) .positioning-region:active {
+      background: ${neutralFillStealthActive};
+      color: ${neutralForegroundActive};
     }
-
-    .positioning-region::before {
-        content: "";
-        display: block;
-        width: var(--tree-item-nested-width);
-        flex-shrink: 0;
-    }
-
-    .positioning-region:hover {
-        background: ${neutralFillStealthHover};
+    :host(:${focusVisible}) .positioning-region {
+      background: ${neutralFillStealthFocus}
+      border-color: ${focusStrokeOuter};
+      box-shadow: 0 0 0 calc(${focusStrokeWidth} * 1px) ${focusStrokeOuter} inset;
+      color: ${neutralForegroundHover};
     }
-
-    .positioning-region:active {
-        background: ${neutralFillStealthActive};
+    .positioning-region::before {
+      content: "";
+      display: block;
+      width: var(--tree-item-nested-width);
+      flex-shrink: 0;
     }
-
     .content-region {
-        display: inline-flex;
-        align-items: center;
-        white-space: nowrap;
-        width: 100%;
-        height: calc(${heightNumber} * 1px);
-        margin-inline-start: calc(${designUnit} * 2px + 8px);
-        font-size: ${typeRampBaseFontSize};
-        line-height: ${typeRampBaseLineHeight};
-        font-weight: 400;
+      display: inline-flex;
+      align-items: center;
+      white-space: nowrap;
+      width: 100%;
+      height: calc(${heightNumber} * 1px);
+      margin-inline-start: calc(${designUnit} * 2px + 8px);
+      font-size: ${typeRampBaseFontSize};
+      line-height: ${typeRampBaseLineHeight};
+      font-weight: 400;
     }
-
     .items {
-        display: none;
-        /* TODO: adaptive typography https://github.com/microsoft/fast/issues/2432 */
-        font-size: calc(1em + (${designUnit} + 16) * 1px);
+      display: none;
+      ${
+          /* Font size should be based off calc(1em + (design-unit + glyph-size-number) * 1px) -
+            update when density story is figured out */ ""
+      } font-size: calc(1em + (${designUnit} + 16) * 1px);
```

Comment:
Use standard block comments in CSS:

```suggestion
      /* Font size should be based off calc(1em + (design-unit + glyph-size-number) * 1px) -
         update when density story is figured out */
      font-size: calc(1em + (${designUnit} + 16) * 1px);
```

Reply:
fixed